plugins/module_utils/data_set.py

```diff
@@ -300,6 +306,85 @@ def data_set_cataloged(name):
             return True
         return False
 
+    @staticmethod
+    def get_volume_list_for_cataloged_data_set(name):
+        """Get the volume list for a cataloged dataset name.
+
+        Arguments:
+            name (str) -- The data set name to check if cataloged.
+        Returns:
+            list{str} -- A list of volumes where the dataset is cataloged.
+        """
+        name = name.upper()
+        module = AnsibleModuleHelper(argument_spec={})
+        stdin = " LISTCAT ENTRIES('{0}') ALL".format(name)
+        rc, stdout, stderr = module.run_command(
+            "mvscmdauth --pgm=idcams --sysprint=* --sysin=stdin", data=stdin
+        )
+        delimeter = 'VOLSER------------'
```

Comment:
The word should be spelled "delimiter", as in "it limits", not "delimeter", which looks like "it measures".

Reply:
Agree, will work on this.